runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -721,8 +722,8 @@ func s:HandleDisasmMsg(msg)
 
       let lnum = search('^' . s:asm_addr)
       if lnum != 0
-        exe 'sign unplace ' . s:asm_id
-        exe 'sign place ' . s:asm_id . ' line=' . lnum . ' name=debugPC'
+      exe 'sign unplace ' . s:asm_id
+      exe 'sign place ' . s:asm_id . ' line=' . lnum . ' name=debugPC'
```

Comment:
could slap an extra indent for these but it doesn't really matter :P

Reply:
next time, then -- I don't trust these vim-patch branches around local _and_ remote changes ;)